functional_tests.go

```diff
@@ -4912,15 +4967,18 @@ func testComposeObjectErrorCasesWrapper(c *minio.Client) {
 	srcSlice := srcArr[:]
 	dst, err := minio.NewDestinationInfo(bucketName, "object", nil, nil)
 	if err != nil {
-		logError(function, args, startTime, "", "NewDestinationInfo failed", err)
+		logError(testName, function, args, startTime, "", "NewDestinationInfo failed", err)
 		return
 	}
 
+	args["destination"] = dst
+	// Set args["sourceList"] to "" to stop having 10,001 null headers logged
+	args["sourceList"] = ""
```

Comment:
You could say something like "source array of 10001 elements".

Reply:
Done.